utilities/logic/async_wrapper.py

```diff
@@ -48,8 +48,8 @@ def daemonize_self():
             sys.exit(0)
     except OSError:
         e         = get_exception()
-        error_str = 'fork #1 failed: %d (%s)' % (e.errno, e.strerror)
-        module.fail_json(error_str)
+        print >>sys.stderr, "fork #1 failed: %d (%s)" % (e.errno, e.strerror)
```

Comment:
His one still needs to be made into a python3 compatible print.  Since 2.4 doesn't have a print function and python3 doesn't use the >> stderr syntax we have to change it to use sys.stderr:
``` python
sys.stderr.write("fork #1 failed: %d (%s)\n" % (e.errno, e.strerror))
```